includes/compat/tinymce.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * TinyMCE functions for classic editor support.
```

Comment:
`compat` is not really the right place for this.

Let's make this a class in `includes/TinyMCE.php` for now.

Reply:
Added the class and removed file inside `compat`